Userland/Libraries/LibJS/Bytecode/Op.cpp

```diff
@@ -698,6 +699,54 @@ ThrowCompletionOr<void> GetNewTarget::execute_impl(Bytecode::Interpreter& interp
     return {};
 }
 
+ThrowCompletionOr<void> GetImportMeta::execute_impl(Bytecode::Interpreter& interpreter) const
+{
+    // 1. Let module be GetActiveScriptOrModule().
+    auto script_or_module = interpreter.vm().get_active_script_or_module();
+
+    // 2. Assert: module is a Source Text Module Record.
+    VERIFY(script_or_module.has<NonnullGCPtr<Module>>());
+    VERIFY(script_or_module.get<NonnullGCPtr<Module>>());
+    VERIFY(is<SourceTextModule>(*script_or_module.get<NonnullGCPtr<Module>>()));
+    auto& module = static_cast<SourceTextModule&>(*script_or_module.get<NonnullGCPtr<Module>>());
```

Comment:
First, `get<T>()` asserts that the variant `has<T>()`, the first assertion is redundant
the rest:
```suggestion
    auto& module = verify_cast<SourceTextModule>(*script_or_module.get<NonnullGCPtr<Module>>());
```